web/apps/labelstudio/src/assets/icons/index.js

```diff
@@ -26,5 +26,8 @@ export { ReactComponent as LsThumbsUp } from './thumbs-up.svg';
 export { ReactComponent as IconUpload } from './upload.svg';
 export { ReactComponent as LsPencil } from './pencil.svg';
 export { ReactComponent as IconInfoOutline } from './info-outline.svg';
+export { ReactComponent as IconSparkGrey } from './spark_grey.svg'
```

Comment:
Can we rename this to kebab-case? The other surrounding definitions are using that. (locality nit, not a preference of mine 🤣)
```suggestion
export { ReactComponent as IconSparkGrey } from './spark-grey.svg'
```